script/bootstrap

```diff
@@ -8,17 +8,25 @@
 
 
 cd "$(dirname "${0}")/.."
-source services.ini
+eval "$(cat services.ini  | ./ini_parser.py)"
 cd services
 top=$(pwd)
+
 mkdir -p log
-for service in $SERVICES; do
+
+for repo in ${SERVICES[@]}; do
+    IFS="@"
+    set $repo
+    service=$1
+    version=$2
+
     cd $top
-    echo "Cloning $service"
-    git clone -q -o github github.com:dod-ccpo/$service
+    echo "Cloning $service at $version"
+    curl -sSL https://github.com/dod-ccpo/$service/archive/$version.tar.gz | tar xvz
```

Comment:
Also, I'd say leave out the `v` option to cut down on the noise (or redirect the output to a log file)